### PR TITLE
fix the field

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue # .

### Summary of Changes
- only allow selected images to be passed in the field
- remove the mootools code from the script
- remove bind polyfill

### Testing Instructions
Try creating an article and selecting intro and full image. Everything should work apart the preview (related to bootstrap popover, bug on their side)

### CODE REVIEW

**Comment by dgrammatiko** [comment]
@laoneo @dneukirchen we have to figure out what we're going to do with the xtd-image

**Comment by laoneo** [comment]
Please open an issue for it 😉

**Comment by laoneo** [comment]
Thanks

**Comment by dneukirchen** [comment]
And we need to support the url param (e_name, author, asset ... ) in the client.

**Comment by dneukirchen** [comment]
Rest looks fine for me.

File: media/system/js/fields/mediafield.js [hunk 1]
```diff
@@ -65,14 +40,26 @@
...
 			self.modalClose();
 			return false;
 		});
 
 
 		window.document.addEventListener('onMediaFileSelected', function (e) {
-			self.selectedPath = e.item.path;
+			var path = e.item.path;
+
+			if (path.match(/.jpg|.jpeg|.gif|.png/)) {
```
Comment (dneukirchen) [inline media/system/js/fields/mediafield.js:57]: Would be nice to find a better, more flexible and less hardcoded way for this.

**Pull request merged**